### DISCUSSION TITLE
How to handle installation - with Comodo Firewall

### DISCUSSION THREAD

**Discussion by LukySwiss** [winsiderss/systeminformer#1839] [Questions / Answers] [answered]
After using processhacker I found that the project now is devolped under systeminformer.
So I downloaded the latest knightly setup.exe.

Running the setup.exe in Windows 10 did ask me if I want to install this software from unknown source.
And Comodo FW recommended to have the program contained (I guess sandbox using VTRoot...).

**Questions**

- Is this because the nightly builds are not signed?
- And if Yes, why is this the case?
- Is it still save to run the nightly builds?
- should I even tell Comodo to have systeminformer setup.exe not contained?

**Remarks**

- Sorry for the stupid questions ... but I better ask anyhow.
- Would be glad to learn more about systeminformer - as this seems the most capable winsys tool tracing issues.
- My goal is letting systeminformer write a log file with amount of data transferred to any IP-address per process ... when I start it. --> feel free to give me a hint.

Thank you very much.
 L.

**Comment by dmex** [comment, marked as answer]
> Is this because the nightly builds are not signed?

Yes.

> why is this the case?

Appveyor and Github don't support HSM (Hardware Security Module) on premise.

> Is it still save to run the nightly builds?

You should be asking Comodo to explain why they're flagging the project as malicious or why they're flagging the project unsafe.
